### ISSUE TITLE
Uno should include content, pages, and resources automatically

### ISSUE THREAD

**Issue by Youssef1313** [unoplatform/uno#14065]
### What would you like to be added

Currently, templates have to have this block:

https://github.com/unoplatform/uno.templates/blob/f62fb25683ab57fe4c3b6b637481d3bce4e0aefc/src/Uno.Templates/content/unoapp/MyExtensionsApp._1/MyExtensionsApp._1.csproj#L267-L273

It would be best if Uno can just do it through the targets in NuGet package. The work for this should include looking into any MSBuild properties used by WinUI and use the same property names to have a matching behavior

### Why is this needed

Build-side parity with WinUI

**Comment by jeromelaban** [comment]
Yes it should. It was not added a while back to avoid having duplicated includes as a backward compatibility fix, and to do this now, we'd need to a way to detect this, with an explicit property, for instance.

**Comment by MartinZikmund** [comment]
Part of Uno.Sdk now
